test/browser/main.fmf

```diff
@@ -64,5 +64,6 @@
     - targetcli
     - udisks2-lvm2
     - udisks2-iscsi
+    - udisks2-btrfs
```

Comment:
That package doesn't exist on CentOS/RHEL, so it'll have to be installed programmatically in browser.sh: https://artifacts.dev.testing-farm.io/27910a8c-ef36-4f46-b11c-6ae33a6b4875/

Reply:
Ah, interesting. browser.sh already installs it, but for the "basic" plan... hmmm.